src/rtnl/link/nlas/bond_port.rs

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: MIT
+use anyhow::Context;
+use byteorder::{ByteOrder, NativeEndian};
+use netlink_packet_utils::{
+    nla::{DefaultNla, Nla, NlaBuffer},
+    parsers::{parse_i32, parse_u16, parse_u32, parse_u8},
+    traits::Parseable,
+    DecodeError,
+};
+
+use crate::constants::*;
+
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+#[non_exhaustive]
+pub enum SlaveState {
+    ACTIVE,
+    BACKUP,
+    Other(u8),
+}
+
+impl From<u8> for SlaveState {
+    fn from(value: u8) -> Self {
+        use self::SlaveState::*;
+        match value {
+            IFLA_BOND_SLAVE_STATE_ACTIVE => ACTIVE,
+            IFLA_BOND_SLAVE_STATE_BACKUP => BACKUP,
+            _ => Other(value),
+        }
+    }
+}
+
+impl From<SlaveState> for u8 {
+    fn from(value: SlaveState) -> Self {
+        use self::SlaveState::*;
+        match value {
+            ACTIVE => IFLA_BOND_SLAVE_STATE_ACTIVE,
+            BACKUP => IFLA_BOND_SLAVE_STATE_BACKUP,
+            Other(other) => other,
+        }
+    }
+}
+
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+#[non_exhaustive]
+#[allow(non_camel_case_types)]
+pub enum MiiStatus {
+    UP,
+    GOING_DOWN,
+    DOWN,
+    GOING_BACK,
```

Comment:
Please follow the naming scheme of rust: https://rust-lang.github.io/api-guidelines/naming.html

Reply:
The reason that I did follow before is to be consistent with iproute2, https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/tree/ip/iplink_bond_slave.c?h=v6.3.0#n44.

Reply:
I followed the naming scheme of rust. Thanks.